spec/controllers/v2/security_guides_controller_spec.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe V2::SecurityGuidesController do
+  let(:security_guides) { FactoryBot.create_list(:v2_security_guide, 20) }
+
+  context 'Unauthorized' do
+  end
+
+  context 'Authorized' do
+    let(:current_user) { FactoryBot.create(:v2_user) }
+
+    before(:each) do
+      request.headers['X-RH-IDENTITY'] = current_user.account.identity_header.raw
+      expect(controller).to receive(:rbac_allowed?).and_return(rbac_allowed?)
+    end
+
+    describe 'GET index' do
+      context 'RBAC allowed' do
+        let(:rbac_allowed?) { true }
+
+        it 'responds with ok status' do
+          get :index
+
+          expect(response).to have_http_status :ok
+        end
+
+        it 'returns security guides' do
+          sg = security_guides.first
+
+          get :index
+
+          expected = {
+            'id': sg.id,
+            'ref_id': sg.ref_id,
+            'version': sg.version
+          }
+
+          expect(response.parsed_body).to include(expected)
+        end
+
+        it 'paginates' do
+          PER_PAGE = 10
+
+          get :index, params: { 'limit' => PER_PAGE }
+
+          binding.pry
+
+          expect(response.parsed_body['data'].count).to eq(PER_PAGE)
+        end
```

Comment:
Do you really test pagination here? It seems to me that you're just checking if the limit parameter works properly.